test/integration/scheduler/framework_test.go

```diff
@@ -1309,6 +1306,8 @@ func TestCoSchedulingWithPermitPlugin(t *testing.T) {
 		waitReject bool
 		waitAllow  bool
 	}{
+		// TODO Separate these two subtests into two tests so they do not share the test scheduler.
```

Comment:
That's not exactly what we want.
You could have a single `Test`, but just init the scheduler inside the loop.

Reply:
You can just leave the implementation details out of the TODO. Simply say: "make the subtests not share scheduler instances"